.travis.yml

```diff
@@ -1,20 +0,0 @@
-after_success:
-  - npx codecov
-  - npx codeclimate-test-reporter < ./coverage/lcov.info
-cache:
-  directories:
-    - node_modules
-language: node_js
-node_js:
-  - node
-  - 8
-notifications:
-  webhooks:
-    urls:
-      - "https://webhooks.gitter.im/e/94a8a87b117fbc50ceaf"
```

Comment:
I think this can be supported via `Settings > Webhooks`

Reply:
This is super old the Gitter rooms are removed so you can disregard this.